RELEASE-LEADS.md

```diff
@@ -119,104 +109,30 @@ Make a copy of the
 empty it out and send it to the WG leads of the respective project (serving or
 eventing) to fill in. Coordinate with both serving and eventing leads.
 
-### Cut `release-x.y` in `test-infra`, `pkg`, `caching`, and `networking` libraries
-
-Shared dependencies like `knative/{test-infra, pkg, caching, networking}` are
-kept up-to-date nightly in each of the releasing repositories. To stabilize
-things shortly before the release we cut the `release-x.y` branches on those 7
-days prior to the main release.
-
-First, create a release branch for `test-infra` named `release-x.y`.
-
-Next, `pkg` needs to pin to `test-infra`'s release branch. To do that, edit
-`hack/update-deps.sh` in `pkg` **on the newly created branch** to pin the
-branch. Then run `./hack/update-deps.sh --upgrade` and commit the changes.
-
-The change to `hack/update-deps.sh` will look like this:
-
-```diff
-diff --git a/hack/update-deps.sh b/hack/update-deps.sh
-index a39fc858..0634362f 100755
---- a/hack/update-deps.sh
-+++ b/hack/update-deps.sh
-@@ -26,7 +26,7 @@ cd ${ROOT_DIR}
- # The list of dependencies that we track at HEAD and periodically
- # float forward in this repository.
- FLOATING_DEPS=(
--  "knative.dev/test-infra@master"
-+  "knative.dev/test-infra@release-x.y"
- )
+### Cut release branches of supporting repos
 
- # Parse flags to determine any we should pass to dep.
-```
+We need to start cutting release branches in each module that does not produce a
+release artifact, but we will do it from least dependent to most dependent.
+Follow the [cutting release branches](#cutting_release_branches) guide, starting
```

Comment:
The `cutting release branches` seems to be a dead link.

Reply:
fixed: `#cutting-release-branches`